CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## 2.0.2
 
+* Eliminate the use of barback (which is now deprecated). This change
+  requires many changes in the implementation, but it should not give
+  rise to incompatibilities for clients of reflectable.
+
+* Update many version dependencies: TODO
+
+## 2.0.2
```

Comment:
It seems this version now appears twice in the changelog.

Reply:
Corrected to 2.0.3, which is a suitable version number for pub.dartlang.org as well.